extensions/pqr/manifest.json

```diff
@@ -0,0 +1,1113 @@
+        "License": "MIT + file LICENSE",
+        "URL": "https://github.com/yihui/xfun",
+        "BugReports": "https://github.com/yihui/xfun/issues",
+        "Encoding": "UTF-8",
+        "RoxygenNote": "7.3.2",
+        "VignetteBuilder": "litedown",
+        "NeedsCompilation": "yes",
+        "Packaged": "2025-08-18 23:51:54 UTC; yihui",
+        "Author": "Yihui Xie [aut, cre, cph] (ORCID:\n    <https://orcid.org/0000-0003-0645-5666>, URL: https://yihui.org),\n  Wush Wu [ctb],\n  Daijiang Li [ctb],\n  Xianying Tan [ctb],\n  Salim Brüggemann [ctb] (ORCID: <https://orcid.org/0000-0002-5329-5987>),\n  Christophe Dervieux [ctb]",
+        "Maintainer": "Yihui Xie <xie@yihui.name>",
+        "Repository": "RSPM",
+        "Date/Publication": "2025-08-19 02:50:02 UTC",
+        "Built": "R 4.4.0; x86_64-pc-linux-gnu; 2025-08-19 05:02:53 UTC; unix"
+      }
+    },
+    "yaml": {
+      "Source": "CRAN",
+      "Repository": "https://p3m.dev/cran/latest",
+      "description": {
+        "Package": "yaml",
+        "Type": "Package",
+        "Title": "Methods to Convert R Data to YAML and Back",
+        "Date": "2024-07-22",
+        "Version": "2.3.10",
+        "Suggests": "RUnit",
+        "Author": "Shawn P Garbett [aut], Jeremy Stephens [aut, cre], Kirill Simonov [aut], Yihui Xie [ctb],\n  Zhuoer Dong [ctb], Hadley Wickham [ctb], Jeffrey Horner [ctb], reikoch [ctb],\n  Will Beasley [ctb], Brendan O'Connor [ctb], Gregory R. Warnes [ctb],\n  Michael Quinn [ctb], Zhian N. Kamvar [ctb], Charlie Gao [ctb]",
+        "Maintainer": "Shawn Garbett <shawn.garbett@vumc.org>",
+        "License": "BSD_3_clause + file LICENSE",
+        "Description": "Implements the 'libyaml' 'YAML' 1.1 parser and emitter\n  (<https://pyyaml.org/wiki/LibYAML>) for R.",
+        "URL": "https://github.com/vubiostat/r-yaml/",
+        "BugReports": "https://github.com/vubiostat/r-yaml/issues",
+        "NeedsCompilation": "yes",
+        "Packaged": "2024-07-22 15:44:18 UTC; garbetsp",
+        "Repository": "RSPM",
+        "Date/Publication": "2024-07-26 15:10:02 UTC",
+        "Encoding": "UTF-8",
+        "Built": "R 4.4.0; x86_64-pc-linux-gnu; 2024-07-27 06:09:28 UTC; unix"
+      }
+    }
+  },
+  "files": {
+    ".Rprofile": {
+      "checksum": "2d09ca1fa1be8cbf725dfa4d74d0bb2a"
+    },
+    "index.qmd": {
+      "checksum": "e3c9e1907f7e77e8d7da0492e92d9def"
+    },
+    "README.md": {
+      "checksum": "238e109b5ca384ff1d1df76ffe2c0bb8"
+    },
+    "renv.lock": {
+      "checksum": "09128737486c3529550c51992aba60d0"
+    },
+    "requirements.in": {
+      "checksum": "a8339a5379053005e03306a5b9792ffb"
+    },
+    "requirements.txt": {
+      "checksum": "0a2d1b4f222461c5c172b2fc227aed0d"
+    }
+  },
+  "users": null,
+  "extension": {
+    "name": "pqr",
+    "title": "pqr",
```

Comment:
Going off naming of other Gallery items:

```suggestion
    "title": "Quarto Document with Python and R",
```

Not sure if Document is the right word, but its more descriptive. 

<img width="1149" height="263" alt="Image" src="https://github.com/user-attachments/assets/b7e1aa87-ce49-4488-9d1a-694e3dcfa6c6" />